data/maps/zone1/act1/entities/interactables.csv

```diff
@@ -0,0 +1,233 @@
+Adv2,INTERACTABLES,78,12,,,,,
```

Comment:
With the header, what I meant was the first line could be a column header to help with readability of what columns mean what?

Reply:
I suppose that's hard when this first line is for understanding the contents of the file?

Reply:
I *could* change it so that the first line is a "data descriptor" and the "header" gets moved to the second, but I didn't want to have a copy-paste of it in every single file, so I decided against it.

It's not particularly good for manual editing anyway. I'm rather seeing it as a format that'll be easier to use when making an editor than the binary format